pipelines/cellpainting/cpd_max_projection_pipeline.wdl

```diff
@@ -15,23 +15,28 @@ workflow cpd_max_projection_distributed {
     # Specify input file information, images directory & extension
     String images_directory_gsurl
     String? file_extension = ".tiff"
+    String load_data_directory_gsurl
 
     # Specify Metadata used to distribute the analysis: Well (default), Site..
     String splitby_metadata = "Metadata_Well"
 
     # And the desired location of the outputs
     String output_directory_gsurl
+    String output_load_data_directory_gsurl = output_directory
 
   }
 
   # Ensure paths do not end in a trailing slash
   String images_directory = sub(images_directory_gsurl, "/+$", "")
+  String load_data_directory = sub(load_data_directory_gsurl, "/+$", "")
   String output_directory = sub(output_directory_gsurl, "/+$", "")
+  String output_load_data_directory = sub(output_load_data_directory_gsurl, "/+$", "")
+
 
   # Create an index to scatter
   call util.scatter_index as idx {
     input:
-      load_data_csv= images_directory + "/load_data.csv",
+      load_data_csv= load_data_directory + "/load_data.csv",
```

Comment:
and then changing this to 
```wdl
load_data_csv=load_data_gsurl,
```

what do you think?

Reply:
That way, just in case somebody wanted to call their file `load_data(1).csv` or something, it would still work and allow that

Reply:
Incorporated in latest commit